python/paddle/fluid/layers/tensor.py

```diff
@@ -364,6 +365,27 @@ def zeros(shape, dtype, force_cpu=False):
     return fill_constant(value=0.0, **locals())
 
 
+def scatter(input, index, updates):
```

Comment:
I think the scatter layer can be generated automatically through `generate_layer_fn`.